src/Domain/CoordinateMaps/FocallyLiftedFlatSide.cpp

```diff
@@ -0,0 +1,304 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/FocallyLiftedFlatSide.hpp"
+
+#include <array>
+#include <cmath>
+#include <limits>
+#include <optional>
+#include <pup.h>
+#include <type_traits>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Parallel/PupStlCpp11.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/DereferenceWrapper.hpp"
+#include "Utilities/EqualWithinRoundoff.hpp"
+#include "Utilities/ErrorHandling/Assert.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TypeTraits/RemoveReferenceWrapper.hpp"
+
+namespace domain::CoordinateMaps::FocallyLiftedInnerMaps {
+
+FlatSide::FlatSide(const std::array<double, 3>& center,
+                   const double inner_radius,
+                   const double outer_radius) noexcept
+    : center_(center),
+      inner_radius_(inner_radius),
+      outer_radius_(outer_radius) {
+  ASSERT(not equal_within_roundoff(inner_radius, 0.0),
+         "Cannot have zero radius.  Note that this ASSERT implicitly "
+         "assumes that the radius has a scale of roughly unity.  Therefore, "
+         "this ASSERT may trigger in the case where we intentionally want "
+         "an entire domain that is very small.  If we really want small "
+         "domains, then this ASSERT should be modified.");
+  ASSERT(outer_radius > inner_radius,
+         "Radii in wrong order: " << inner_radius << " " << outer_radius);
```

Comment:
maybe `"Outer radius is not larger than inner radius. Inner radius:" << inner_radius << " outer raidus: " << outer_radius`?